BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -247,7 +234,40 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
         attentionView.valueLabel.text = "\(info.percent)%"
       }
     }
-
+  }
+  
+  private func updatePublisherView(completion: (() -> Void)?) {
+    guard let host = state.url.host else { return }
+    self.state.ledger.publisherInfo(forId: host) { [weak self] info in
```

Comment:
`publisherInfo(forId:)` is not actually asynchronous, and the block is non-escaping on the Obj-C++ side, so you can technically remove the `completion` block

I'm going to add `NS_NOESCAPE` to blocks to the Obj-C++ side that aren't actually escaping or even update said APIs to not use completion blocks if I can so its more obvious that we can use the results of the block synchronously

Reply:
Fixed.